app/models/chat_room.rb

```diff
@@ -0,0 +1,19 @@
+# == Schema Information
+#
+# Table name: chat_rooms
+#
+#  id            :bigint           not null, primary key
+#  title         :string           not null
+#  user_owner_id :bigint           not null
+#  user_guest_id :bigint           not null
+#  created_at    :datetime         not null
+#  updated_at    :datetime         not null
+#
+
+class ChatRoom < ActiveRecord::Base
+  belongs_to :user_owner, class_name: :User, foreign_key: 'user_owner_id'
+  belongs_to :user_guest, class_name: :User, foreign_key: 'user_guest_id'
```

Comment:
I would prefer a `has_many :users` relation. This way you avoid `own_chat_rooms ` and `guest_chat_rooms` in the user model.

Reply:
I know @gabrielbursztein2 that's a better approach but for the purpose of this induction, I think it's enough as it is.